src/js/veteran-id-card/containers/VeteranIDCard.jsx

```diff
@@ -42,6 +57,8 @@ const mapStateToProps = (state) => {
     profile: userState.profile,
     loginUrl: userState.login.loginUrl,
     verifyUrl: userState.login.verifyUrl,
+    serviceRateLimitedAuthed: Math.random() > rateLimitAuthed,
```

Comment:
@ncksllvn curious why in https://github.com/department-of-veterans-affairs/vets-website/pull/6906/files for RequiredLoginView.jsx you moved this out of the constructor and into defaultProps. Thinking this could go in `componentDidMount()` instead, right?

Reply:
Yeah it may be better have it determined in `componentDidMount()` since that's where the rest of the logic is, or maybe into `mapStateToProps`. At the time I didn't really see a reason to keep it in state, because that value is only calculated once (since after page load that component will only be rendered at most one time) and then should remain static, so defaultProps gives us that while also exposing that value as something we could add unit tests for or manipulate from a parent component if we wanted to.